requirements/reproducible-mac.txt

```diff
@@ -695,17 +704,12 @@ sarif-om==1.0.4 \
     --hash=sha256:539ef47a662329b1c8502388ad92457425e95dc0aaaf995fe46f4984c4771911 \
     --hash=sha256:cd5f416b3083e00d402a92e449a7ff67af46f11241073eea0461802a3b5aef98
     # via cfn-lint
-serverlessrepo==0.1.10 \
```

Comment:
Hello, thank you for your contribution.

Some days ago, I was thinking:

- "why `aws-sam-cli` is utilizing `serverlessrepo` considering the current status (only-read | public archive)?"

I have some questions now:

 - How do you make sure that the package `serverlessrepo` is not been in use?;
 - Is there any other package with this same behaviour? (unused).
 
 Seems to me that another non-used package soon or later "could" lead to another breaking. 👋

Reply:
Hi @realFranco, the `serverlessrepo` package was in fact being used by the `sam publish` command. We brought it in to be vendored by us yesterday https://github.com/aws/aws-sam-cli/pull/5572. All other dependencies are being used.

Reply:
Nice, thank you for the clarification.